wpeverest/everest-forms
# Tweak - Styles for progress bar

### All Submissions:






### Changes proposed in this Pull Request:



This PR adds styles for the component progress bar.

### How to test the changes in this Pull Request:

1. To test this PR go to the `enhancement/password-bar` branch on the Everest Forms Pro.
2. Check the progress bar while using the password field.

### Types of changes:




### Other information:




### Changelog entry

> Add - Styles for the progress bar.
